src/scripts/app.ts

```diff
@@ -379,11 +379,19 @@ export class ComfyApp {
     const paintedIndex = selectedIndex + 1
     const combinedIndex = selectedIndex + 2
 
+    // for vueNodes mode
+    let images = node.images
```

Comment:
I don't like that we have `imgs` and `images` in the same scope.

Reply:
Side-nit: Could you use a nullish coalescing operator to not need the conditional or a mutable reference here?